### PR TITLE
Try out preliminary support for Python 3.9

### PR DESCRIPTION
**Pull request by icemac**
There are failing tests in some packages which need to be fixed before we are able to support Python 3.9.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,8 +18,8 @@ before_install:
     - pip install -U setuptools
 install:
     - pip install -U six==$(awk '$1 == "six" {print $3}' ztk-versions.cfg)
-    - python bootstrap.py
-    - bin/buildout -n
+    - pip install -U zc.buildout==$(awk '$1 == "zc.buildout" {print $3}' ztk-versions.cfg)
```
Comment (mgedmin) [inline .travis.yml]: Does `-U` make sense when you're pinning an explicit version?

Comment (icemac) [reply .travis.yml]: You are right, as `zc.buildout` is not yet installed `-U` does not matter.

**Comment by icemac** [comment]
There are failing tests on Python 3.9 for:

- [ ] zope.container
- [ ] zope.keyreference

**Comment by jamadden** [comment]
Probably because of https://github.com/zopefoundation/persistent/issues/124

**Comment by icemac** [comment]
@jamadden Thank you for fixing `persistent`. It helped to fix the tests here, too.

**Pull request merged**